### PR TITLE
Feature/home carousel slides

### PR DESCRIPTION
**Pull request by weberjavi**
This PR adds slide images to home page according to https://projects.invisionapp.com/d/main#/console/15803555/332182849/inspect

Images have been pre processed with https://imageoptim.com
To automate this flow worth exploring https://github.com/Klathmon/imagemin-webpack-plugin or similar 

kapture 2018-11-20 at 16 40 38

Note slide text content is yet to be defined.

### CODE REVIEW

File: app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx [hunk 1]
```diff
@@ -1,48 +1,115 @@
...
+  },
+  {
+    pagingTitle: 'Top 10 emitting provinces',
+    title: 'Top 10 emitting provinces',
+    buttonText: 'Go to historical emissions',
+    smImage: provincesSmImage,
+    bgImage: provincesBgImage,
+    altText: 'Provinces chart',
+    redirectAction: CLIMATE_GOALS
+  }
```
Comment (aabdaab) [inline app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx]: do we know if this data will be coming from data files or it's gonna be manageable from admin panel? If so, we can wait until we merge translations branch and we add slugs for these to fetch content from BE

**Comment by aabdaab** [comment]
another thing would be to update the links of the buttons in the carousel in `National Context` and `Top 10...` cause they both redirect to `Climate goals`

File: app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx (hunk 1 above)
Comment (weberjavi) [reply app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx]: Not really sure about this one, perhaps @simaob knows better.

Comment (aabdaab) [reply app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx]: Doesn't look like sth coming from data files. Translations are already on develop, so now you can rebase and rely on content coming from BE :) more info here ---> https://github.com/ClimateWatch-Vizzuality/indonesia-platform/pull/17 
or just ping me when in doubts :D

**Comment by weberjavi** [comment]
Ready to review again ☝️

**Comment by weberjavi** [comment]
@aabdaab, @Bluesmile82 ready ☝️ with the content being feed from translations `.yml` file

**Pull request merged**